sippy-ng/src/prow_job_runs/ProwJobRun.js

```diff
@@ -60,12 +60,24 @@ export default function ProwJobRun(props) {
 
   const fetchData = () => {
     let queryString = ''
-    console.log('hello world we got the prow job run id of ' + props.jobRunID)
+    console.log(
+      'We got the prow job run id of ' + props.jobRunID,
+      props.jobName
+        ? `jobName of ${props.jobName}`
+        : '' + props.pullNumber
+        ? `pullNumber of ${props.pullNumber}`
+        : '' + props.repoInfo
+        ? `repoInfo of ${props.repoInfo}`
+        : ''
+    )
```

Comment:
Please find a  simpler way to log this, it's fine if we print "jobName=" if there was no jobName for example.

Reply:
The pretty printer insists on making it look ugly.

This:

```js
      props.jobName
        ? `jobName of ${props.jobName}`
        : '' + props.pullNumber
```

should be this:

```js
      props.jobName ? `jobName of ${props.jobName}` : '' + props.pullNumber
```

But, I like your suggestion to simplify with "jobName=" style.